homeassistant/components/flume/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "flume",
   "name": "flume",
   "documentation": "https://www.home-assistant.io/integrations/flume/",
-  "requirements": ["pyflume==0.2.4"],
+  "requirements": ["pyflume==0.3.0"],
```

Comment:
This version doesn't exist on PyPI https://pypi.org/project/PyFlume/#history

Reply:
Ah https://github.com/ChrisMandich/PyFlume/pull/2

Reply:
@balloob  @ChrisMandich was able to get 0.3.0 out :)